cogs/utils/tibia.py

```diff
@@ -178,41 +178,39 @@ def him_her(self) -> str:
                 log.info(f"get_character(): {old_name} renamed to {character.name}")
                 bot.dispatch("character_rename", character, old_name)
 
-        # Discord owner
-        db_char = await conn.fetchrow("""SELECT id, user_id, name, user_id, vocation, world, guild, sex
-                                        FROM "character"
-                                        WHERE name LIKE $1""", character.name)
+        # Get character in database
+        db_char = await DbChar.get_by_name(conn, character.name)
         if db_char is None:
             # Untracked character
             return
 
-        character.owner_id = db_char["user_id"]
-        character.id = db_char["id"]
+        character.owner_id = db_char.user_id
+        character.id = db_char.id
         _vocation = character.vocation.value
-        if db_char["vocation"] != _vocation:
-            await conn.execute('UPDATE "character" SET vocation = $1 WHERE id = $2', _vocation, db_char["id"])
-            log.info(f"get_character(): {character.name}'s vocation: {db_char['vocation']} -> {_vocation}")
+        if db_char.vocation != _vocation:
+            await db_char.update_vocation(conn, _vocation, False)
```

Comment:
If all commands have `update_self` passing `False`, why isn't that the default? Are you planning future changes where default `True` makes more sense?

Reply:
Gonna merge it for now, as this is a minor point and can be easily removed.

The solutions are:
- Make the default False, but this would make the method a bit counter intuitive.
- Change the implementations, to save a reference to the previous value